lib/matplotlib/__init__.py

```diff
@@ -311,7 +384,32 @@ def ge(self, level):
         return self.vald[self.level] >= self.vald[level]
 
 
-verbose = Verbose()
+def _wrap(fmt, func, level='INFO', always=True):
+    """
+    return a callable function that wraps func and reports it
+    output through logger
+
+    if always is True, the report will occur on every function
+    call; otherwise only on the first time the function is called
```

Comment:
I don't think we ever use always=True so we may as well drop that

Reply:
The whole class is deprecated, so...